src/main/java/uk/gov/justice/hmpps/prison/service/AppointmentsService.kt

```diff
@@ -125,7 +124,7 @@ class AppointmentsService(
   @Transactional
   fun createBookingAppointment(
     bookingId: Long,
-    username: String,
+    username: String?,
```

Comment:
previously `username` would actually hold the principal, hence couldn't be `null`.  Since now holds the `username` it needs to be nullable.